### DISCUSSION TITLE
Possibly to make the exported Word file read-only?

### DISCUSSION THREAD

**Discussion by affans** [quarto-dev/quarto-cli#3808] [Q&A] [answered]
I am using the `vscode` extension to export a `qmd` file to a `docx` file. Everything works great but I'd rather have the extension produce a read-only file so that others should only modify the `qmd` file if they want changes. 

Is it possible to set some sort of parameter to make the file read only? It seems like it is possible since the 'preview' when exporting to word file shows a readonly file.

**Comment by mcanouil** [comment, marked as answer]
Create a post-render script where you set the permission you want to the file.
Note that the permission does not matter if the user is root or as root privileges.

See <https://quarto.org/docs/projects/scripts.html#pre-and-post-render>.

PS: please change the category from "general" to "Q&A" and if appropriate mark this as the answer to let others know it has been answered.
